docs/configure-artifact-repository.md

```diff
@@ -554,3 +554,12 @@ configuring the default artifact repository described previously.
       command: [sh, -c]
       args: ["cp -r /my-input-artifact /my-output-artifact"]
 ```
+
+## Artifact Streaming
+
+With artifact streaming, artifacts don’t need to be saved to disk first. Artifact streaming is only supported in the following
+artifact drivers: S3 (v3.4+), HTTP (v3.5+), and Artifactory (v3.5+).
```

Comment:
I see it here too: https://github.com/argoproj/argo-workflows/blob/master/workflow/artifacts/azure/azure.go#L208

Reply:
Added!